src/raftstore/store/worker/apply.rs

```diff
@@ -209,64 +216,87 @@ impl ApplyDelegate {
             }
 
             let res = match entry.get_entry_type() {
-                EntryType::EntryNormal => self.handle_raft_entry_normal(entry),
-                EntryType::EntryConfChange => self.handle_raft_entry_conf_change(entry),
+                EntryType::EntryNormal => self.handle_raft_entry_normal(&mut wb, &mut cbs, entry),
+                EntryType::EntryConfChange => {
+                    self.handle_raft_entry_conf_change(&mut wb, &mut cbs, entry)
+                }
             };
 
             if let Some(res) = res {
                 results.push(res);
             }
         }
 
+        if !self.pending_remove {
+            rocksdb::get_cf_handle(&self.engine, CF_RAFT)
```

Comment:
L199 has comment for `pending_remove`.
I think it is clearly that if this peer is remove itself, no need write status to rocksdb.